app/controllers/development_controller.rb

```diff
@@ -173,34 +174,20 @@ def setup_busy_discussion_with_signed_in_user
     redirect_to discussion_url(test_discussion)
   end
 
-  def setup_group_on_trial_admin
-    group_on_trial = Group.new(name: 'Ghostbusters', is_visible_to_public: true)
-    GroupService.create(group: group_on_trial, actor: patrick)
-    membership = Membership.find_by(user: patrick, group: group_on_trial)
-    membership.experienced! 'welcomeModal'
-    group_on_trial.add_member! jennifer
-    sign_in patrick
-    redirect_to group_url(group_on_trial)
-  end
-
-  def setup_group_on_trial
-    GroupService.create(group: test_group, actor: patrick)
-    sign_in jennifer
-    redirect_to group_url(test_group)
-  end
-
-  def setup_group_with_expired_trial
-    GroupService.create(group: test_group, actor: patrick)
-    subscription = test_group.subscription
-    subscription.update_attribute :expires_at, 1.day.ago
+  def setup_group_on_free_plan
+    group = Group.new(name: 'Ghostbusters',
+                      is_visible_to_public: true)
+    GroupService.create(group: group, actor: patrick)
+    membership = Membership.find_by(user: patrick, group: group)
+    group.add_member! jennifer
     sign_in patrick
-    redirect_to group_url(test_group)
+    redirect_to group_url(group)
   end
 
-  def setup_group_with_overdue_trial
+  def setup_group_and_select_plan
+    test_group.segments['bx_choose_plan'] = true
```

Comment:
Yea.. cool noticing. I'm on the fence.. maybe slightly in favour of combining them actually. It seems like ab tests are about experiences... and if we were to do this.. then we don't have more BX specific schema on the models, and we get to extend experiences to Group.